sdk/communication/communication-networktraversal/test/public/utils/matrix.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
```

Comment:
good news, this class has now been moved into a common package so you won't need your own version of it
Please see @azure/test-utils